tests/visual/percy-storybook/Style.percy.tsx

```diff
@@ -374,4 +388,124 @@ storiesOf('DashTable/Style type condition', module)
+            },
+            backgroundColor: 'MediumPurple'
+        }]}
+    />))
+    .add('header header_index array', () => (<DataTable
+        setProps={setProps}
+        id='table'
+        {...DEFAULT_TABLE}
+        columns={DEFAULT_TABLE.columns.map(c => ({ ...c, name: [c.name, c.name, c.name] }))}
+        style_header_conditional={[
+            { if: { header_index: [0, 2] }, background_color: 'blue', color: 'white' }
+        ]}
+    />))
+
+    .add('cell column_id array', () => (<DataTable
+        {...DEFAULT_TABLE}
+        id='data-column-id-array'
+        style_cell_conditional={[{
+            if: {
+                column_id: ['a', 'b', 'd']
+            },
+            backgroundColor: 'MediumPurple'
+        }]}
+    />))
+    .add('filter column_id array', () => (<DataTable
+        {...DEFAULT_TABLE}
+        id='data-column-id-array'
+        filter_action='native'
+        style_filter_conditional={[{
+            if: {
+                column_id: ['a', 'b', 'd']
+            },
+            backgroundColor: 'MediumPurple'
+        }]}
+    />))
+    .add('header column_id array', () => (<DataTable
+        {...DEFAULT_TABLE}
+        id='data-column-id-array'
+        style_header_conditional={[{
+            if: {
+                column_id: ['a', 'b', 'd']
+            },
+            backgroundColor: 'MediumPurple'
+        }]}
+    />))
+    .add('active styling', () => (<DataTable
+        {...DEFAULT_TABLE}
+        id='active-styling'
+        style_data_conditional={[{
+            if: {
+                state: 'active'
+            },
+            backgroundColor: 'lightblue',
+            border: '1px solid blue'
+        }]}
+        active_cell={{ row: 1, column: 1 }}
+    />))
+    .add('selected styling (applied to active)', () => (<DataTable
+        {...DEFAULT_TABLE}
+        id='data-column-id-array'
+        style_data_conditional={[{
+            if: {
+                state: 'selected'
+            },
+            backgroundColor: 'lightblue',
+            border: '1px solid blue'
+        }]}
+        selected_cells={[
+            { row: 1, column: 1, column_id: 'b' },
+            { row: 1, column: 2, column_id: 'c' },
+            { row: 2, column: 1, column_id: 'b' },
+            { row: 2, column: 2, column_id: 'c' }]}
+        active_cell={{ row: 1, column: 1 }}
+    />))
+    .add('active styling overrides selected', () => (<DataTable
+        {...DEFAULT_TABLE}
+        id='data-column-id-array'
+        style_data_conditional={[
+            {
+                if: {
+                    state: 'selected'
+                },
+                backgroundColor: 'lightblue',
+                border: '1px solid blue'
+            },
+            {
+                if: {
+                    state: 'active'
+                },
+                backgroundColor: 'black',
+                color: 'white',
+                border: '1px solid white'
+            }
```

Comment:
Can we add one where `active` _partially_ overrides `selected`? Like:
```js
{
  if: {state: 'selected'},
  backgroundColor: 'lightblue',
  border: '1px solid blue'
},
{
  if: {state: 'active'},
  color: 'MediumPurple',
  border: '2px solid black'
}
```
In that case, the active cell would still have a light blue background, right?

Reply:
Yes, that's the expected behavior. https://github.com/plotly/dash-table/pull/729/commits/8085950b3ffbc3c618db19df52a314640387a4ce